components/com_content/src/Model/CategoryFormModel.php

```diff
@@ -0,0 +1,501 @@
+                    $where[] = $db->quoteName('key') . ' = :oldKey';
+                    $query->bind(':oldKey', $oldKey);
+                }
+
+                $query->extendWhere('AND', $where, 'OR');
+            }
+
+            $db->setQuery($query);
+
+            try {
+                $db->execute();
+            } catch (\RuntimeException $e) {
+                $this->setError($e->getMessage());
+
+                return false;
+            }
+
+            if (!$allLanguage) {
+                $associations[$table->language] = (int) $table->id;
+            }
+
+            if (\count($associations) > 1) {
+                $key = md5(json_encode($associations));
+                $query->clear()
+                    ->insert($db->quoteName('#__associations'))
+                    ->columns(
+                        [
+                            $db->quoteName('id'),
+                            $db->quoteName('context'),
+                            $db->quoteName('key'),
+                        ]
+                    );
+
+                foreach ($associations as $id) {
+                    $id = (int) $id;
+
+                    $query->values(
+                        implode(
+                            ',',
+                            $query->bindArray(
+                                [$id, $this->associationsContext, $key],
+                                [ParameterType::INTEGER, ParameterType::STRING, ParameterType::STRING]
+                            )
+                        )
+                    );
+                }
+
+                $db->setQuery($query);
+
+                try {
+                    $db->execute();
+                } catch (\RuntimeException $e) {
+                    $this->setError($e->getMessage());
+
+                    return false;
+                }
+            }
+        }
+
+        Factory::getApplication()->triggerEvent($this->event_after_save, [$context, &$table, $isNew, $data]);
+
+        if (!$table->rebuildPath($table->id)) {
+            $this->setError($table->getError());
+
+            return false;
+        }
+
+        if (!$table->rebuild($table->id, $table->lft, $table->level, $table->path)) {
+            $this->setError($table->getError());
+
+            return false;
+        }
+
+        $this->setState($this->getName() . '.id', $table->id);
+
+        if (Factory::getApplication()->getInput()->get('task') == 'editAssociations') {
+            return $this->redirectToAssociations($data);
+        }
+
+        $this->cleanCache();
+
+        return true;
+    }
+
+    /**
+     * Method to get a table object, load it if necessary.
+     *
+     * @param   string  $type    The table name. Optional.
+     * @param   string  $prefix  The class prefix. Optional.
+     * @param   array   $config  Configuration array for model. Optional.
+     *
+     * @return  \Joomla\CMS\Table\Table  A Table object.
+     *
+     * @since   6.1.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

doing this means our build scripts will correctly set the version - it might not make it to 6.1 ;)